helper_lib.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+ps_command=$(command -v ps)
+ps_args='-U root -u root -f'
```

Comment:
Since the Docker daemon needs to be run as root, there's no need to try to parse all user processes imho.